### ISSUE TITLE
plugin description file for RainbowText v1.1 is invalid

### ISSUE THREAD

**Issue by sphings79** [Sord1d/RainbowText#2]
```
[15:04:17 ERROR]: Permission node 'rainbowtext.use' in plugin description file for RainbowText v1.1 is invalid
java.lang.IllegalArgumentException: Data cannot be null
	at org.apache.commons.lang.Validate.notNull(Validate.java:192) ~[commons-lang-2.6.jar:2.6]
	at org.bukkit.permissions.Permission.loadPermission(Permission.java:294) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.permissions.Permission.loadPermissions(Permission.java:241) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.PluginDescriptionFile.getPermissions(PluginDescriptionFile.java:868) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugin(CraftServer.java:553) ~[paper-1.18.1.jar:git-Paper-187]
	at org.bukkit.craftbukkit.v1_18_R1.CraftServer.enablePlugins(CraftServer.java:478) ~[paper-1.18.1.jar:git-Paper-187]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:727) ~[paper-1.18.1.jar:git-Paper-187]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:503) ~[paper-1.18.1.jar:git-Paper-187]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:313) ~[paper-1.18.1.jar:git-Paper-187]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1202) ~[paper-1.18.1.jar:git-Paper-187]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-187]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[15:04:17 INFO]: [RainbowText] Enabling RainbowText v1.1
```

Paper 1.18.1 latest build and your latest from Spigot

**Comment by sord1d** [comment]
It seems that you're using an outdated version. This bug should be fixed in version 1.2:
[https://github.com/Sord1d/RainbowText/releases/tag/1.2](https://github.com/Sord1d/RainbowText/releases/tag/1.2
)
Could you try that build and give some feedback whether the issue still exists?

**Comment by sphings79** [comment]
Thx works like charm.
Don't know why i doesn't get an update notification from spigot
